scripts/litePageSizeValidator/index.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-console */
+const { exec } = require('child_process');
+
+const MAX_PAGE_SIZE_KB = 100;
+
+const litePageSizeValidator = async () => {
+  const urlsToCheck = [
+    { path: '/hindi', pageType: 'Home' },
+    { path: '/mundo/articles/cddylv9g8z0o', pageType: 'Optimo Article' },
+    {
+      path: '/nepali/bbc_nepali_radio/liveradio',
+      pageType: 'Live Radio',
+    },
+    {
+      path: '/arabic/media-53135426',
+      pageType: 'Media Asset with Live Stream',
```

Comment:
Apologies this should be 

```suggestion
      pageType: 'CPS Media Article with Live Stream',
```

I just inspected the HTML of https://www.bbc.com/arabic/media-53135426 and I can see `MediaArticlePage` chunks